core-bundle/src/DependencyInjection/Attribute/AsCronJob.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\DependencyInjection\Attribute;
+
+/**
+ * An attribute to register a Contao cron job.
+ */
+#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::IS_REPEATABLE)]
+class AsCronJob
+{
+    public function __construct(public string $interval, public ?string $method = null)
+    {
```

Comment:
The CronJob service annotation also has the following logic:

https://github.com/contao/contao/blob/cf615b4640f0a00c1e2a19f5604cb1b86a6d8c4f/core-bundle/src/ServiceAnnotation/CronJob.php#L40-L41

Reply:
I don't think we need to escape values in attributes anymore?

Reply:
Yeah, this was only necessary for annotations since they are in PHP comments. You cannot write

```php
/**
 * @CronJob(*/5 * * *)
 */
```

for example, since `*/` closes the comment. You need to escape the `*` or `/`:

```php
/**
 * @CronJob(\*/5 * * *)
 */
```

With PHP attributes this is (presumably) not necessary.